lib/etrade/parse-etrade-gl.ts

```diff
@@ -0,0 +1,72 @@
+import {
+  GainAndLossEvent,
+  GainAndLossEventXlsxRow,
+  PlanType,
+} from "./etrade.types";
+import { getDateString } from "./get-date-string";
+import XLSX from "xlsx";
+
+const toDateString = (rawDate: string) =>
+  getDateString(new Date(Date.parse(rawDate)));
+
+export const parseEtradeGL = async (
+  file: File
+): Promise<GainAndLossEvent[]> => {
+  const data: GainAndLossEvent[] = [];
+  const fileAsArrayBuffer = await file.arrayBuffer();
+  const workbook = XLSX.read(fileAsArrayBuffer);
+  const worksheet = workbook.Sheets[workbook.SheetNames[0]];
+  const rawData = XLSX.utils.sheet_to_json(worksheet, { header: 2 });
+  // First row is summary
+  for (let rowIdx = 1; rowIdx < rawData.length; rowIdx++) {
+    const row = rawData[rowIdx] as GainAndLossEventXlsxRow;
+    try {
+      data.push({
+        planType: row["Plan Type"],
+        quantity: row["Qty."],
+        proceeds: row["Proceeds Per Share"],
+        adjustedCost: row["Adjusted Cost Basis Per Share"],
+        dateAcquired: toDateString(row["Date Acquired"]),
+        dateSold: toDateString(row["Date Sold"]),
+        isPlanUsQualified: row["Qualified Plan"] === "Qualified",
+        // For now consider that non-US qualified plan is FR
+        // qualified.
+        isPlanFrQualified: row["Qualified Plan"] !== "Qualified",
```

Comment:
the `Qualified Plan` column doesn't exist in the collapsed version of the spreadsheet. we should update the guide to require the expanded spreadsheet.

Reply:
Fixed.
Though it will fail gracefully for now as you don't rely on the export to check if the plan is qualified but on the user input.